onnx/defs/nn/defs.cc

```diff
@@ -3754,44 +3763,46 @@ ONNX_OPERATOR_SET_SCHEMA(
             builder.Add("present_value = Identity (PresentValue)");
           }
 
-          // If attn_mask is provided
+          builder.Add("NewKVSeqLen =  Shape <start = -2, end = -1> (PresentKey)");
+          builder.Add("AttnBiasShape = Concat <axis = -1> (QSeqLen, NewKVSeqLen)");
```

Comment:
I think we can use `axis = 0`, right? Seems preferable, since it can be interpreted without knowing the rank of input tensors (so downstream tools will likely handle it better).

Reply:
Right. Updated.